<?xml version="1.0"?>
<launch>
  <arg name="dataRoot" value="$(env HOME)" />
  <arg name="tabletop_segmentation_service" value="tabletop_segmentation" />
  <arg name="map_file" default="$(find pr2_pbd_interaction)/config/map.yaml"/>

  <include file="$(find pr2_social_gaze)/launch/gaze.launch" />
  <include file="$(find pr2_moveit_config)/launch/move_group.launch" />
  <include file="$(find tabletop_object_detector)/launch/tabletop_segmentation.launch"/>

  <node name="pr2_pbd_interaction" pkg="pr2_pbd_interaction" type="interaction.py" output="screen">
    <param name="dataRoot" value="$(arg dataRoot)" />
    <param name="actionsRoot" value="$(arg dataRoot)/pbd_actions" />
    <param name="isReload" value="false" />
    <param name="experimentNumber" value="0" />
    <param name="tabletop_segmentation_service" value="$(arg tabletop_segmentation_service)" />
  </node>

  <!-- Run the map server -->
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />
</launch>
